lib/Interpreter/IncrementalParser.cpp

```diff
@@ -280,6 +280,55 @@ namespace cling {
     return m_Consumer->getTransaction();
   }
 
+  const Transaction*
+  IncrementalParser::mergeTransactionsAfter(const Transaction *T, bool Prev) {
```

Comment:
I am looking at the latest. I expect to see
```
assert(!T->isNestedTransaction() && "Stem of transaction merge must be a topmost transaction.");
```
(or a nicer worded version of the same) at the top of the function.